src/components/Blocking/ReportDomain.js

```diff
@@ -139,7 +144,8 @@ const styles = StyleSheet.create({
   container: {
     height: '80%',
     borderTopRightRadius: 20,
-    borderTopLeftRadius: 20
+    borderTopLeftRadius: 20,
+    backgroundColor: COLORS.almostBlack
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color of the container has been changed to `COLORS.almostBlack`. This change should be reviewed to ensure it matches the desired aesthetic of the application.

```diff
-    borderTopLeftRadius: 20
+    borderTopLeftRadius: 20,
+    backgroundColor: COLORS.almostBlack
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->